packages/graphql/src/translate/queryAST/ast/operations/composite/CompositeReadPartial.ts

```diff
@@ -62,8 +61,16 @@ export class CompositeReadPartial extends ReadOperation {
             // NOTE: This is slightly different to ReadOperation for cypher compatibility, this could use `WITH *`
             matchClause.where(wherePredicate);
         }
-        const subqueries = Cypher.concat(...this.getFieldsSubqueries(nestedContext));
-        const sortSubqueries = wrapSubqueriesInCypherCalls(nestedContext, this.sortFields, [targetNode]);
+        // const subqueries = Cypher.concat(...this.getFieldsSubqueries(nestedContext));
+        // const sortSubqueries = wrapSubqueriesInCypherCalls(nestedContext, this.sortFields, [targetNode]);
+
+        const cypherFieldSubqueries = this.getCypherFieldsSubqueries(nestedContext);
+        const subqueries = Cypher.concat(...this.getFieldsSubqueries(nestedContext), ...cypherFieldSubqueries);
+        // const subqueries = Cypher.concat(...this.getFieldsSubqueries(nestedContext));
+        const sortSubqueries = this.sortFields
```

Comment:
There's a lot of leftover comments here. Was that intended? :thinking:

Reply:
Nope, sorry I missed those